variables.tf

```diff
@@ -67,6 +67,12 @@ variable "iam_partition" {
   description = "IAM Partition (Select aws-us-gov for AWS GovCloud, otherwise leave as is)"
 }
 
+variable "metadata_service_container_image" {
+  type = string
+  default = "netflixoss/metaflow_metadata_service:2.2.2"
```

Comment:
A bit of a nitpick, but I wonder if there is a way we can avoid setting this default value in 3 places? If we can somehow make the top module to use whatever the default in the submodule, that would at least make it 2 places instead of 3.